app/src/main/java/com/fbu/fbuteam/SignUpActivity.java

```diff
@@ -0,0 +1,84 @@
+package com.fbu.fbuteam;
```

Comment:
I would suggest rewriting it as: https://gist.github.com/LeoNero/9bb5fb07d8f544a44c118edbef3e3921